packages/react-jsx-highmaps/src/index.js

```diff
@@ -0,0 +1,15 @@
+export * from 'react-jsx-highcharts';
+
+// Charts
+export { default as HighchartsMapChart } from './components/HighchartsMapChart';
+
+// Graph Parts
+export { default as MapNavigation } from './components/MapNavigation';
+export { default as XAxis } from './components/XAxis';
+export { default as YAxis } from './components/YAxis';
```

Comment:
XAxis and YAxis are already exported by:
```javascript
export * from 'react-jsx-highcharts';
```
Might be better to name these as MapXAxis and MapYAxis, so they don't get confused with the highcharts ones. It seems that they behave differently than the ones from react-jsx-highcharts, but I haven't tested.

Reply:
I think I might remove `export * from 'react-jsx-highcharts';` and export only the parts that are relevant to maps. Unlike Highstock, I don't think people should be trying to build charts with this package.

Reply:
That might be better. Could also be:
```javascript
export { withHighcharts as withHighmaps } from 'react-jsx-highcharts';
```